images/specification.excalidraw

```diff
@@ -0,0 +1,1014 @@
+{
```

Comment:
I wonder if we need the source as well?

Reply:
Source? Do you mean links to the specs?

Reply:
Oh I mean the excalidraw file.

Reply:
The advantage is that is allows someone to easily update the image the next time the spec changes. What do you reckon? I'm ok either way.

Reply:
@adityasaky shall I resolve this comment? Assuming everything else is okay to get this approved?

Reply:
I'm not one of the @in-toto/attestation-maintainers, I'm fine though with adding the source file as well.

Reply:
Yes that's the idea.